WordPressKit/CommentServiceRemoteREST.m

```diff
@@ -453,7 +452,7 @@ - (RemoteComment *)remoteCommentFromJSONDictionary:(NSDictionary *)jsonDictionar
     comment.authorUrl = jsonDictionary[@"author"][@"URL"];
     comment.authorAvatarURL = [jsonDictionary stringForKeyPath:@"author.avatar_URL"];
     comment.commentID = jsonDictionary[@"ID"];
-    comment.content = jsonDictionary[@"content"];
+    comment.content = jsonDictionary[@"raw_content"];
```

Comment:
Hmm. 

When the context is `display` (the default) `content` should contain rendered output. Any short codes or whatever would be converted to HTML.  Consuming `raw_content` preserves existing functionality but I wonder if we might have a use for the rendered `content` as well.  I'm not sure (and we can always add it later). Thoughts?

Maybe leave a comment explaining why we're assigning raw_content to the `content` param to help avoid any future confusion?

Reply:
Done! Thanks!